services/provider/server/server.go

```diff
@@ -937,13 +941,15 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					}),
 				},
 				&pb.ExternalResource{
-					Name: "cephfs",
+					Name: fmt.Sprintf("%s-groupsnapclass", req.StorageClaimName),
```

Comment:
How is it matching with the internal mode, as it does not include `ceph-rbd` in the name? Are we relying on the claim name to have that in its name?

Reply:
The default storageClaim name for internal modes are "ocs-storagecluster-ceph-rbd" and "ocs-storagecluster-cephfs"